com.google.AndroidStudio.json

```diff
@@ -0,0 +1,78 @@
+{
+    "app-id": "com.google.AndroidStudio",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "android",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--socket=system-bus",
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=home",
+        "--allow=multiarch",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "git",
+            "make-args": ["NO_PERL=1"],
+            "make-install-args": ["NO_PERL=1"],
+            "cleanup": ["/share/git-gui"],        
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.15.0.tar.xz", 
+                    "sha256": "107116489f10b758b51af1c5dbdb9a274917b0fb67dc8eaefcdabc7bc3eb3e6a"
+                }
+            ]
+        },
+        {
+            "name": "android-studio",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D apply_extra /app/bin",
+                "install -D android.sh /app/bin/android",
+                "install -Dm644 com.google.AndroidStudio.svg /app/share/share/icons/hicolor/512x512/apps/com.google.AndroidStudio.svg",
```

Comment:
Looks like the build is failing because this icon is not found; it should be added as a `file` source below similar to how it's done for the `appdata.xml`.

Reply:
yeah, stupid error on my part

Reply:
fixed now.